version/version.go

```diff
@@ -22,14 +22,15 @@ import (
 
 const (
 	// OperatorVersionString is the version string of this operator binary.
-	OperatorVersionString = "1.3.0"
+	OperatorVersionString = "1.3.1"
```

Comment:
You may as well update this to 1.4.0.

Reply:
Fixed.

Reply:
Changes not pushed?

Reply:
Will update to 1.4.0. This was reverted by my local testing code.
When using 1.4.0, it will get error "translateConfig/translateConfig-1.4.yaml not found."

Reply:
translateConfig-1.4.yaml added.